access-mgmt/README.adoc

```diff
@@ -0,0 +1,469 @@
+====
+
+// Purpose statement for the assembly
+[role="_abstract"]
+As an owner of a Kafka instance in {product}, you can manage the level of access that other user accounts and service accounts have to your instance. You can allow or deny operations that specific accounts or that all accounts in your organization can perform for your instance or for certain resources in that instance, such as topics. You can also allow other users or service accounts to manage access to the resources in your instance for you.
+
+You can manage access for only the Kafka instances that you create or for instances that the owner has enabled you to access and alter.
+
+//Additional line break to resolve mod docs generation error, not sure why. Leaving for now. (Stetson, 20 May 2021)
+
+[id="con-access-management_{context}"]
+== Access management in {product}
+
+[role="_abstract"]
+{product} uses Access Control Lists (ACLs) provided by Kafka that enable you to manage how other user accounts and service accounts are permitted to interact with the Kafka resources that you create. You can manage access for only the Kafka instances that you create or for instances that the owner has enabled you to access and alter.
+
+A resource in an ACL can be a Kafka instance, topic, consumer group, or producer transaction. A user account enables users in your organization to access your resources. A service account enables your application or tool to connect securely to your instance and access your resources.
+
+You can define ACLs for the following components:
+
+* A single account or all accounts within the organization
+* A single named resource, all resources of a particular type (such as a topic, consumer group, or transactional ID), or all resources of a particular type with a specified prefix
+* A single operation (such as `Write`) for a resource or all operations for a resource
+
+You can also allow other users or service accounts to manage access to the resources in your instance for you.
+
+If two or more permission settings in an ACL match a request being made to the Kafka broker and at least one of the matching permissions specifies that the action is denied, then the request is denied.
+
+[role="_additional-resources"]
+.Additional resources
+* link:https://kafka.apache.org/documentation/#security_authz[Authorization and ACLs^] in Kafka documentation
+
+=== Supported ACL permissions in {product}
+
+An ACL acts as a mapping of permitted operations on specified resources for a selected account or for all accounts in an organization. An account can be a user account or a service account. Operations correspond to Kafka APIs or request types that relate to the specified resource.
```

Comment:
Ah, here is a little more context about what "account" means. Might be useful to put this up a bit higher and clarify before users get down here?

Reply:
Yep, see above.